challenge_14/c/karanchawla/challenge_14.c

```diff
@@ -0,0 +1,90 @@
+//Karan Chawla
+//Challenge 14
+
+#include <stdio.h>
+#include <stdlib.h>
+
+//Linked list node definition
+typedef struct node
+{
+	char c;
+	struct node* next;
+}Node;
+
+
+//utility function to create a new node
+Node* newNode(char c)
+{
+	Node* NewNode = (Node*)malloc(sizeof(Node));
+
+	NewNode->c = c;
+	NewNode->next = NULL;
+
+	return NewNode;
+}
+
+//utility function to add a new node to the linked list
+void push(Node** head_ref, char c)
+{
+	Node* new_node = newNode(c);
+
+	new_node->next = (*head_ref);
```

Comment:
I'm curious, why are you wrapping the de-references in parenthesis? It isn't needed and looks a little odd.